extended-statistics.md

```diff
@@ -0,0 +1,163 @@
+
+## Common operations
+
+### Enable extended statistics
+
+To enable extended statistics, set the system variable `tidb_enable_extended_stats` to `ON`:
+
+```sql
+SET GLOBAL tidb_enable_extended_stats = ON;
+```
+
+The default value of this variable is `OFF`. This setting is a one-time task.
+
+### Register extended statistics
+
+The registration for extended statistics is not a one-time task, and you need repeat the registration for each extended statistics.
+
+To register extended statistics, use the SQL statement `ALTER TABLE ADD STATS_EXTENDED`. The syntax is as follows:
+
+```sql
+ALTER TABLE table_name ADD STATS_EXTENDED IF NOT EXISTS stats_name stats_type(column_name, column_name...);
+```
+
+In the syntax, you can specify the table name, statistics name, statistics type, and column name of the extended statistics to be collected.
+
+- `table_name` specifies the name of the table from which the extended statistics are collected.
+- `stats_name` specifies the name of the statistics, which must be unique for each table.
+- `stats_type` specifies the type of the statistics. Currently, only the correlation type is supported.
+- `column_name` specifies the column group, which might have multiple columns. Currently, you can only specify two column names.
+
+<details>
+<summary> How it works</summary>
+
+To improve access performance, each TiDB node maintains a cache in the system table `mysql.stats_extended` for extended statistics. After you register the extended statistics, the next time the `ANALYZE` statement is executed, TiDB will collect the extended statistics if the system table `mysql.stats_extended` has the corresponding records.
+
+Each row in the `mysql.stats_extended` table has a `version` column. Once a row is updated, the value of `version` is increased. In this way, TiDB loads the table into memory incrementally, instead of fully.
+
+TiDB loads `mysql.stats_extended` periodically to ensure that the cache is kept the same as the data in the table.
+
+> **Warning:**
+>
+> It is **NOT RECOMMENDED** to directly operate on the `mysql.stats_extended` system table. Otherwise, inconsistent caches occur on different TiDB nodes.
+>
+> If you have mistakenly operated on the table, you can use the following statement on each TiDB node. Then the current cache will be cleared and the `mysql.stats_extended` table will be fully reloaded:
+>
+> ```sql
+> ADMIN RELOAD STATS_EXTENDED;
+> ```
+
+</details>
+
+### Delete extended statistics
+
+To delete a record of the extended statistics, use the following statement:
+
+```sql
+ALTER TABLE table_name DROP STATS_EXTENDED stats_name;
+```
+
+<details>
+<summary>How it works</summary>
+
+After you execute the statement, TiDB marks the value of the corresponding record in `mysql.stats_extended`'s column `status` to `2`, which means that the record is deleted, instead of deleting the record directly.
```

Comment:
For the word "record", how about using "object" instead? @winoros @TomShawn If "object" is not appropriate, we can also use "record" in all places, including those preceding places where I suggest adding "object" to make sentences complete.

Reply:
According to discussion, "object" is OK.